nuxeo-core/nuxeo-core-management/src/main/java/org/nuxeo/ecm/core/management/probes/StreamProbe.java

```diff
@@ -18,41 +18,96 @@
  */
 package org.nuxeo.ecm.core.management.probes;
 
+import java.time.Duration;
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+
+import org.nuxeo.runtime.api.Framework;
 import static org.nuxeo.lib.stream.computation.log.ComputationRunner.GLOBAL_FAILURE_COUNT_REGISTRY_NAME;
 import static org.nuxeo.lib.stream.computation.log.ComputationRunner.NUXEO_METRICS_REGISTRY_NAME;
 
 import org.nuxeo.ecm.core.management.api.Probe;
 import org.nuxeo.ecm.core.management.api.ProbeStatus;
-
+import org.nuxeo.runtime.services.config.ConfigurationService;
 import com.codahale.metrics.Counter;
 import com.codahale.metrics.MetricRegistry;
 import com.codahale.metrics.SharedMetricRegistries;
 
 /**
- * A probe to detect when computation has been terminated due to failure.
+ * A probe to detect when computation has been terminated due to failure. A
```

Comment:
Is you IDE still configured with a line length of 80?
Comparing the Javadoc wit the one from [master](https://github.com/nuxeo/nuxeo/commit/acf91b4fe144a1481caa38ee090b03bff8bfd133#diff-e733524112971085719b9e47b54b9fa1R39).

Reply:
my line wrapping is set to 120, I don't know why it wraps at 80 ...

Reply:
my line wrapping is set to 120, I don't know why it wraps at 80 ...